dubbo-cluster-extensions/dubbo-cluster-specify-address-dubbo2/src/main/java/org/apache/dubbo/rpc/cluster/specifyaddress/Address.java

```diff
@@ -17,10 +17,15 @@
 package org.apache.dubbo.rpc.cluster.specifyaddress;
 
 import org.apache.dubbo.common.URL;
+import org.apache.dubbo.rpc.cluster.common.SpecifyAddress;
 
 import java.io.Serializable;
 import java.util.Objects;
 
+/**
+ * Use {@link SpecifyAddress} instead.
+ */
+@Deprecated
 public class Address implements Serializable {
```

Comment:
extend `org.apache.dubbo.rpc.cluster.common.SpecifyAddress`

Reply:
Or move this class to common module without change its package

Reply:
URL attributes are always cast when used, using Object instead of generics so that they can be placed in the common package